### ISSUE TITLE
Stable version

### ISSUE THREAD

**Issue by felipemorais** [addyosmani/basket.js#49]
Which features are missing to launch a stable version?

**Comment by sindresorhus** [comment]
Copy pasted from a comment in another issue:
- Used in the wild (obviously) - and proven to be stable and pretty bug free.
- Perf tests!
- Complete test suite
- Finalized API
- Case studies
- Docs
- Superlicious website

In addition I would say we would need to resolve the current open issues.

We do however intend to release versions between that.

**Comment by sindresorhus** [comment]
0.3 is tagged and published on Bower.

**Comment by peol** [comment]
Awesome. Good work team!

**Comment by felipemorais** [comment]
Very Nice! :)

**Comment by addyosmani** [comment]
Excellent work all :)

**Comment by sindresorhus** [comment]
@addyosmani Should we announce it somewhere, or should we wait until a more mature release?

**Comment by addyosmani** [comment]
I think we should announce on twitter. Wednesday sound good?

**Comment by peol** [comment]
Sounds good. Just ping this issue before so we can help :)

Mvh,
Andrée

On 1 dec 2012, at 16:47, Addy Osmani notifications@github.com wrote:

I think we should announce on twitter. Wednesday sound good?

—
Reply to this email directly or view it on
GitHubhttps://github.com/addyosmani/basket.js/issues/49#issuecomment-10918427.

**Comment by sindresorhus** [comment]
Sure

**Comment by addyosmani** [comment]
Looks like Wednesday slipped. Monday sound good to everyone? We can tweet out about the release and link to a changelog.

**Comment by peol** [comment]
Yeah, thought about it on tuesday but toally forgot on wednesday. :p

Monday sounds good to me.

**Comment by sindresorhus** [comment]
:+1:

**Comment by peol** [comment]
So... It slipped again, should we aim for a release next Monday so we have time to update the docs regarding using promises now?

**Comment by sindresorhus** [comment]
Hah, we're are hopeless... :p

**Comment by sindresorhus** [comment]
Why monday? I would like for us to try to finish up the docs and get it out today or tomorrow if possible.

**Comment by peol** [comment]
Fine by me, just thought it was open source and all, I won't have time to
help out until this weekend for example :/

**Comment by sindresorhus** [comment]
k, let's see when we get the docs ready and are happy with it ;)

**Comment by sindresorhus** [comment]
Just wanted to let you all know Basket.js was featured on BadassJS!

http://badassjs.com/post/40850339601/basket-js-a-javascript-loader-with-localstorage-based

// @wibblymat

**Comment by addyosmani** [comment]
Saw this a few days ago, but that's really awesome. I'm glad to see this getting some mainstream attention :)

**Comment by sindresorhus** [comment]
@addyosmani felt good being ahead for once? :P  (internal joke)

**Comment by addyosmani** [comment]
@sindresorhus I feel like the happiest man alive. the happiest!

**Comment by felipemorais** [comment]
Awesome!
